evm-forge/.github/scripts/rename.sh

```
@@ -0,0 +1,38 @@
+
+# Overwrite package.json
+echo "$JQ_OUTPUT" > package.json
+
+# Make sed command compatible in both Mac and Linux environments
+# Reference: https://stackoverflow.com/a/38595160/8696958
+sedi () {
+  sed --version >/dev/null 2>&1 && sed -i -- "$@" || sed -i "" "$@"
+}
+
+# Rename instances of "PaulRBerg/foundry-template" to the new repo name in README.md for badges only
+sedi "/gitpod/ s|PaulRBerg/foundry-template|"${GITHUB_REPOSITORY}"|;" "README.md"
```

Shellcheck

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)